cms/wp/README.md

```diff
@@ -0,0 +1,48 @@
+# WordPress Deployer Config
+
+## Example WordPress Configuration
+
+The following will configure Deployer to deploy `git@repository.example:foo/wp`
+to `foo.dev.example`, `foo.staging.example` & `production.example`. It will
+expect that `example-staging` is a [defined host](https://www.ssh.com/academy/ssh/config#format-of-ssh-client-config-file-ssh_config).
```

Comment:
I don't see that repository or any of the host names above (including `example-staging`) referenced anywhere in this section - but maybe I'm missing something?

Reply:
The documentation has been updated in a follow-up commit. I am marking this conversation as resolved, in favor of any follow-up discussion that may need to occur given the new changes.